### ISSUE TITLE
wrong editing link if cockpit is in subfolder

### ISSUE THREAD

**Issue by raffaelj** [pauloamgomes/Cockpit-MultipleCollectionLink#2]
I edit an entry and my url looks like:
`http://localhost/cockpit-next/collections/entry/collectionName/longId`
If I click the edit button, it links to:
`http://localhost/collections/entry/anotherCollection/anotherId`.
The location of cockpit `http://localhost/cockpit-next/...` got lost.

I will send a pull request soon, but I have to test my current solution first.

**Comment by GuillermoCasanova** [comment]
I can second this issue. 

What I did in the meantime is to manually place my website link in the edit link with http:// and everything like so:

<a class="uk-item-link" target="_blank" href="http://MY-WEBSITE-HERE.net/cms/collections/entry/{ l.link }/{ l._id }" title="{ App.i18n.get('Edit Collection') }" data-uk-tooltip="pos:'bottom'"><i class="uk-icon-edit"></i></a>

Probably not the prettiest solution, would be nice to have something where it pulls the required site/link without needing this. 

@raffaelj  was your solution fruitful?

**Comment by raffaelj** [comment]
I didn't use it again and I think, I have the updated version in my test cockpit. So the issue still exists...
And the relative path was a really bad idea. I digged deeper in the code the last weeks and I understand it much better now. <del>I fixed it with this commit: https://github.com/raffaelj/Cockpit-MultipleCollectionLink/commit/e7253af76145f50159fd967df8ef07eb96691dd8</del>

No, wait. I forgot to copy the changed files before testing. This commit is the right one: https://github.com/pauloamgomes/Cockpit-MultipleCollectionLink/pull/3/commits/0ad970daeb5dc754566b112a657a20825c9a22da
